File: app/src/main/java/br/com/jonathanzanella/myexpenses/chargeable/ChargeableType.java
package br.com.jonathanzanella.myexpenses.chargeable;

public enum ChargeableType {
	ACCOUNT,
	DEBIT_CARD,
	CREDIT_CARD;

	public static ChargeableType getType(String type) {
		for (ChargeableType chargeableType : ChargeableType.values()) {
			if(chargeableType.name().equals(type))
				return chargeableType;
		}
		return null;
	}
}

File: app/src/main/java/br/com/jonathanzanella/myexpenses/chargeable/ChargeableType.kt
package br.com.jonathanzanella.myexpenses.chargeable

enum class ChargeableType {
    ACCOUNT,
    DEBIT_CARD,
    CREDIT_CARD;

    companion object {
        fun getType(type: String): ChargeableType {
            return ChargeableType.values().first { it.name == type }
        }
    }
}